doc/release-notes-1.10.rst

```diff
@@ -67,7 +67,11 @@ Networking
 Bluetooth
 *********
 
-* details ...
+* Multiple qualification-related fixes for Bluetooth Mesh
+* Support for acting as Bluetooth Mesh Friend Node
```

Comment:
Support for the Bluetooth Mesh Friend Node role ?

Reply:
sure, fixed